samcli/lib/utils/managed_cloudformation_stack.py

```diff
@@ -33,6 +33,62 @@ def get(self, key) -> Optional[str]:
             return None
 
 
+def update_stack(
```

Comment:
Are these changes going to affect SAM CLI's managed stack, or they are specific to Pipeline command's managed stack?

Reply:
Yes this is only used in Pipeline command's managed stack.